scvi/inference/inference.py

```diff
@@ -16,6 +16,9 @@ class UnsupervisedTrainer(Trainer):
         :gene_dataset: A gene_dataset instance like ``CortexDataset()``
         :train_size: The train size, either a float between 0 and 1 or and integer for the number of training samples
          to use Default: ``0.8``.
+        :n_epochs_kl_warmup: Number of epochs for linear warmup of KL(q(z)||p(z)) term. After `n_epochs_kl_warmup`,
```

Comment:
q(z |x) || p(z), for less confusion and stick to the paper notations ?

Reply:
Sure